ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productlist/v1/productlist/paginationbar.html

```diff
@@ -14,10 +14,15 @@
 <template data-sly-template.pagination="${@ productList}">
```

Comment:
The pagination bar needs to reflect the changes in made in commit 7496ebf4007081559711f31d9adf7979a94ffda9. What I see here seems to be based on an earlier version.
This applies to the pagination bar of the search results too.

Reply:
Done! I actually pulled all of the pagination related code into a new `Pager` class that is a LOT easier to unit test now in isolation and without a bunch of mocking. Hopefully this works for you!